File: 0062-unique-paths/0062-unique-paths.py
class Solution:
    def uniquePaths(self, m: int, n: int) -> int:
        table = [1 for _ in range(n)]
        
        for i in range(1, m):
            for j in range(1, n):
                table[j] += table[j - 1]
                
        return table[-1]

File: 0062-unique-paths/0062-unique-paths.cpp
class Solution {    
    int curr[100];
public:
    int uniquePaths(int m, int n) {
        fill(&curr[0], &curr[100], 1);
    
        for (int i = 1; i < m; i++) {
            for (int j = 1; j < n; j++) {
                curr[j] += curr[j - 1];
            }
        }
        return curr[n - 1];
    }
};